### ISSUE TITLE
Increase discoverability of projects with issues that need help

### ISSUE THREAD

**Issue by freak4pc** [RxSwiftCommunity/rxswiftcommunity.github.io#72]
This continues discussion of #71, initially started by @pedrommcarrasco.

The TL;DR is - we need a good way to indicate to people if some projects have issues that have issues that need contributions. This helps the projects, but also helps contributors become more involved and pull in more new contributors. 

Here is my proposed solution for this issue:

- Mark issues with needs help and good first issue. This is something you, or anyone else really, can definitely help with in the "moderation" side, if you have the time or just see an issue without a tag. This will also help on the GitHub side of things.
<p align="center"><img width="501" alt="36999164-49f50a6a-208d-11e8-95e8-e5ef818ee683" src="https://user-images.githubusercontent.com/605076/36999779-3388c044-208f-11e8-8d17-94260131bcb8.png"></p>

- On the web side, have the script populate the number of issues as well, and then we can have a colored badge (red, orange, yellow, green) based on the number of open issues that need external help (for example green: 0, yellow, 0-3, orange: 3-10, red: >10, but we can experiment with these numbers later). If colors are scary we can go with some other visual indication :) 

We can also add an explanation on the top of the page so people understand these badges, and they would link directly to the issues that need help (e.g. marked with these tags).

Since most issues don't have these tags used we might want to start with a version that counts _all_ issues for a repo instead of just those with these specific tags, and later migrate to the tag-based solution when those become more common. 

Would appreciate feedback from @RxSwiftCommunity/contributors if any :) 
Shai.
